src/renderer/components/NewProjectModal.tsx

```diff
@@ -11,7 +11,13 @@ import { useDispatch } from 'react-redux';
 import { useState } from 'react';
 import { projectCreated, recentProjectAdded } from '../store/actions';
 import SelectMediaBlock from './SelectMediaBlock';
-import { Project } from '../store/helpers';
+import {
+  AudioFileExtension,
+  MediaFileExtension,
+  Project,
+  VideoFileExtension,
+} from '../store/helpers';
+import { extractFileExtension, getMediaType, makeProject } from '../util';
```

Comment:
In this file these imports are not being used: AudioFileExtension MediaFileExtension VideoFileExtension extractFileExtension getMediaType